src/main/java/org/broadinstitute/hellbender/tools/sv/SVCallRecordUtils.java

```diff
@@ -386,8 +386,8 @@ public static SVCallRecord create(final VariantContext variant, boolean keepVari
         } else {
             contigB = contigA;
             // Force reset of END coordinate
-            if (type.equals(StructuralVariantType.INS)) {
-                positionB = positionA + 1;
+            if (type == GATKSVVCFConstants.StructuralVariantAnnotationType.INS) {
```

Comment:
This was actually causing a bug where `.equals(StructuralVariantType.INS)` is always false because the type is different. Somehow I missed this when switching to use `GATKSVVCFConstants.StructuralVariantAnnotationType` in an earlier PR. Using `==` is safer since it won't compile.

Reply:
The change to END is a correction. In the spec, POS=END for insertions.

Reply:
It does seem a little confusing to have `==` here and `.equals` in other parts of the code like on lines 337 and 349 above. Is there a reason for that? Should we change it everywhere to be consistent?

Reply:
@cwhelan Agreed we should be consistent. I think `==` is a slightly better choice for the type checking so I've changed all the comparisons I could find to that.